cmd/bb_browser/BUILD.bazel

```diff
@@ -1,13 +1,23 @@
 load("@com_github_buildbarn_bb_storage//tools:container.bzl", "container_push_official")
 load("@io_bazel_rules_docker//container:container.bzl", "container_image", "container_layer")
-load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_embed_data", "go_library")
+
+go_embed_data(
+    name = "templates",
+    srcs = glob(["templates/**"]),
+    flatten = True,
+    package = "main",
+    string = True,
+    var = "TemplatesData",
+)
 
 go_library(
     name = "go_default_library",
     srcs = [
         "asset_registry.go",
         "browser_service.go",
         "main.go",
+        ":templates",  # keep
```

Comment:
What are your thoughts on placing a `BUILD` file inside of `templates/` that does this? That way Gazelle does the right thing without using `keep`.